databricks/koalas/frame.py

```diff
@@ -2595,6 +2595,96 @@ def merge(left, right: 'DataFrame', how: str = 'inner',
 
         return DataFrame(joined_table.select(*exprs))
 
+    def append(self, other: 'DataFrame', ignore_index: bool = False,
+               verify_integrity: bool = False, sort: bool = None) -> 'DataFrame':
```

Comment:
I think `sort` should be `False` instead of `None`, even though it's `None` in Pandas API.

Reply:
You're right. As I understand, the reason pandas uses `None` is mainly to avoid backward compatibility issues that we don't have with Koalas. I changed it to `False` with 79119d1.